options/locale/locale_en-US.ini

```diff
@@ -2635,6 +2635,7 @@ members.invite_now = Invite Now
 
 teams.join = Join
 teams.leave = Leave
+teams.view = View
```

Comment:
Imho, move to top-level translation `view` here:

https://github.com/go-gitea/gitea/blob/4539a7b0b4fc6539d67ab386eaac4b9aea82bf53/options/locale/locale_en-US.ini#L111

There is nothing about this word that makes it specific to teams. We need to cut down on such useless translations.